packages/jest-environment-node/src/index.ts

```diff
@@ -66,6 +66,14 @@ class NodeEnvironment implements JestEnvironment {
     if (typeof AbortController !== 'undefined') {
       global.AbortController = AbortController;
     }
+    // Event is global in Node >= 15.4
+    if (typeof Event !== 'undefined') {
```

Comment:
Do you know maybe if there are tests that test other globals from this file?
If so it would make sense to have tests for this logic also.
Other than that, good catch and fix.

Reply:
I could not find any existing tests around making globals available. It's possible I missed them, but I was decently thorough.